src/components/Camera.tsx

```diff
@@ -0,0 +1,66 @@
+import { useRef, useEffect, useState } from "react";
+import "./Camera.css";
+
+export const Camera = () => {
+  const [error, setError] = useState<string | null>(null);
+  const [isEnabled, setIsEnabled] = useState(false);
+  const videoRef = useRef<HTMLVideoElement | null>(null);
+  const streamRef = useRef<MediaStream | null>(null);
+
+  useEffect(() => {
+    if (!videoRef.current) return;
+
+    const startStream = async () => {
+      try {
+        const stream = await navigator.mediaDevices.getUserMedia({
+          video: true,
+        });
+        streamRef.current = stream;
+        videoRef.current!.srcObject = stream;
+      } catch (err) {
+        if (err instanceof Error) {
+          setError(err.name);
+        } else {
+          setError("An unknown error occurred");
+        }
+      }
+    };
+
+    const stopStream = () => {
```

Comment:
stopStream is cleanup function. Just inline it to return statement of the useEffect